test/fixtures/dialects/postgres/insert.sql

```diff
@@ -88,3 +88,12 @@ SELECT
 RETURNING
   test,
   test2;
+
+INSERT INTO
+  baz (state, state_changed_at, instance_id)
+SELECT
+  1, 2, 3
+ON CONFLICT (instance_id) DO UPDATE
+SET
+  state = foo,
+  state_changed_at = bar;
```

Comment:
nit: newline

Reply:
Resolved in [0e015ef](https://github.com/sqlfluff/sqlfluff/pull/6863/commits/0e015ef0565e24e26a3c906bc9c48a33980ab75d).